vms/volunteer/models.py

```diff
@@ -87,7 +87,7 @@ class Volunteer(models.Model):
         blank=True,
         validators=[
             RegexValidator(
-               r'^[0-9a-zA-Z\s\.\-?=#:/_&]+$',
+               r'^(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9]\.[^\s]{2,})+$',

```

Comment:
👋 What all strings did you try with this?

Reply:
Various links , websites  link , strings which contain numbers and most of them worked . Is there some change needed?